ext/simplexml/tests/simplexml_import_dom_basic.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+simplexml_import_dom() - Basic test on invalid nodetype error
+--SKIPIF--
+<?php
+if (!extension_loaded('simplexml')) {
+    exit('Skip - SimpleXML extension not loaded');
+}
+?>
+--FILE--
+<?php
+$x = new stdClass();
+
+var_dump(simplexml_import_dom($x));
+?>
+--EXPECTF--
+Warning: simplexml_import_dom(): Invalid Nodetype to import in %s on line %d
```

Comment:
This will be an exception after my changes, and the arginfo/zpp mismatch will be gone from debug builds.

Reply:
You can now rebase on PHP-8.0, my fix has been merged